dev-util/visual-studio-code/visual-studio-code-1.22.2.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit check-reqs eutils xdg-utils
+
+DESCRIPTION="Microsoft's free and open-source code editor."
+HOMEPAGE="https://code.visualstudio.com/"
+SRC_URI="https://github.com/Microsoft/vscode/archive/${PV}.tar.gz -> ${P}.tar.gz
```

Comment:
Please name the tarball to match 'upstream' GitHub name, i.e. `vscode-${PV}`. You can use MY_P since you're using the same thing in S.